app/views/instruments/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= simple_form_for @instrument do |f| %>
+  <%= f.input :name, label: "Instrument Name" %>
+  <%= f.input :description, label: "Description" %>
+  <%= f.input :address, label: "Address" %>
+  <%= f.input :price, label: "Price" %>
+  <%= f.input :category, label: "Category" %>
+
```

Comment:
If we have the attached photos in model, we should have an upload field here: `<%= f.input :photos, as: :file, input_html: { multiple: true } %>`